<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableSettingsDialog</class>
 <widget class="QDialog" name="TableSettingsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>244</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>337</width>
    <height>244</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string extracomment="Dialog for insertion of the new table settings">Table settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="numRowsVerticalLayout">
     <item>
      <widget class="QLabel" name="numRowsLabel">
       <property name="text">
        <string>&amp;Rows:</string>
       </property>
       <property name="buddy">
        <cstring>numRowsSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numRowsSpinBox">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Number of rows in the new table</string>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="numColumnsVerticalLayout">
     <item>
      <widget class="QLabel" name="numColumnsLabel">
       <property name="text">
        <string>&amp;Columns:</string>
       </property>
       <property name="buddy">
        <cstring>numColumnsSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numColumnsSpinBox">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Number of columns in the table</string>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="tableWidthGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="tableWidthLabel">
       <property name="text">
        <string>&amp;Width:</string>
       </property>
       <property name="buddy">
        <cstring>tableWidthDoubleSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="tableWidthDoubleSpinBox">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Width of the table</string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="tableWidthModeComboBox">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="warningLine">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>32</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
